app/components/file-uploader.js

```diff
@@ -365,71 +330,49 @@ export default Component.extend(Analytics, {
                 action: 'drop',
             };
 
-            if (this.get('newNodeNewFile')) {
-                eventData.label = 'Submit - Drop File, New Node';
-            } else if (this.get('nodeLocked')) {
+            if (this.get('newPreprintFile')) {
+                eventData.label = 'Submit - Drop File, New Preprint';
+            } else if (this.get('preprintLocked')) {
                 eventData.label = `${this.get('editMode') ? 'Edit' : 'Submit'} - Drop File, New Version`;
-            } else {
-                eventData.label = 'Submit - Drop File, Existing Node';
             }
             this.get('metrics')
                 .trackEvent(eventData);
         },
     },
 
+    _uploadedFileUnchanged() {
+        // Returns true if a file of the same name was already uploaded successfully (Add mode),
+        // but perhaps a request to the preprint failed. When attempting to retry the request,
+        // we can pull this uploadedFileId to set as the preprint's primary file instead of
+        // reuploading the file which would 409.
+        if (this.get('preprintLocked')) {
+            return false;
+        }
+        return this.get('uploadedFileId') && (this.get('file.name') === this.get('uploadedFileName'));
+    },
+
     _setUploadProperties(files) {
-        if (this.get('nodeLocked')) { // Edit mode, fetch URL for uploading new version
+        if (this.get('preprintLocked')) { // Edit mode, fetch URL for uploading new version
             this.send('uploadNewVersionUrl', files);
         } else { // Add mode, fetch URL for uploading new file
             this.send('uploadNewFileUrl', files);
         }
         this.callback.resolve(this.get('file'));
     },
 
-    _createNewProject(node) {
-        this.set('node', node);
-        this.getContributors(node);
-        this.send('upload');
-        this.set('newNode', true);
-        this.set('applyLicense', true);
+    _sendToUploadPreprint() {
+        this.send('uploadToPreprint');
     },
 
-    _failCreateNewProject() {
+    _failCreateNewPreprint() {
         this.set('uploadInProgress', false);
-        this.get('toast').error(this.get('i18n').t('components.file-uploader.could_not_create_project'));
-    },
-
-    _createComponent(child) {
```

Comment:
File uploader no longer creates a project or component.  The only thing the file uploader does is handle creating a preprint and uploading the file to the preprint, or uploading a new version to the preprint.